src/main/java/org/gridsuite/modification/server/entities/ModificationEntity.java

```diff
@@ -0,0 +1,64 @@
+/*
+  Copyright (c) 2021, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.entities;
+
+import javax.persistence.*;
+
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.util.UUID;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.dto.ModificationInfos;
+
+/**
+ * @author Slimane Amar <slimane.amar at rte-france.com>
+ */
+@NoArgsConstructor
+@Getter
+@Entity
+@Inheritance(strategy = InheritanceType.JOINED)
+@Table(name = "modification")
+public class ModificationEntity {
+
+    public static final String DATE_COLUMN_NAME = "date";
```

Comment:
why single out this column name ?

Reply:
Used in a previous version to sort the modifications
I'm going to delete it

Reply:
Done